projects/charon/src/app/core/services/menu/menu.service.ts

```diff
@@ -130,7 +130,8 @@ export class MenuService extends MenuBaseService {
       map(([user, pdvValue]) => ({
         pdvValue,
         action: () => this.router.navigate(['/', AppRoute.User]),
-        title: `${user.firstName} ${user.lastName}`,
```

Comment:
so, u decided to copypaste `{{ userItem.firstName }} {{ userItem.lastName | slice : 0 : 1 }}.` within the template instead of setting title here